cmd/cloudx/testhelpers/testmain.go

```diff
@@ -1,23 +1,48 @@
 package testhelpers
 
 import (
+	"fmt"
+	"github.com/ory/x/cmdx"
 	"os"
+	"runtime/debug"
 	"testing"
-
-	"github.com/ory/x/stringsx"
 )
 
-func RunAgainstStaging(m *testing.M) {
-	// Run the tests
-	result := m.Run()
-
-	// Use staging
-	if err := os.Setenv("ORY_CLOUD_CONSOLE_URL",
-		stringsx.Coalesce(os.Getenv("ORY_CLOUD_CONSOLE_URL"),
-			"https://project.console.staging.ory.dev")); err != nil {
```

Comment:
Previously we first ran the test, and then swichted to staging. So of course everything ran against prod :sweat_smile:

Reply:
whoops